Description: Change the default of tunable_listen.
 Change the default of tunable_listen to the same as in
 man page vsftpd.conf.
Author: Jörg Frings-Fürst <debian@jff-webhosting.net>
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=783077
Last-Update: 2015-04-21
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: trunk/tunables.c
===================================================================
--- trunk.orig/tunables.c
+++ trunk/tunables.c
@@ -182,7 +182,7 @@ tunables_load_defaults()
   tunable_use_localtime = 0;
   tunable_check_shell = 1;
   tunable_hide_ids = 0;
-  tunable_listen = 1;
+  tunable_listen = 0;
   tunable_port_promiscuous = 0;
   tunable_passwd_chroot_enable = 0;
   tunable_no_anon_password = 0;
